### DISCUSSION TITLE
Jinja2 version

### DISCUSSION THREAD

**Discussion by horizon-mm** [DeniseCaiLab/ezTrack#51] [Show and tell]
The newest version of Jinja2 doesn't work with the installation config described in Wiki, possibly due to the fact that Jinja2_3.1.1 has change the way to import Markup.

pip install Jinja2==3.0.3 can fix that.

**Comment by ZachPenn** [comment]
Thanks for posting this.  Another possible option is to include the jinja2 version in the conda install instructions:
```
conda create -y -n ezTrack -c conda-forge python=3.8 pandas=1.3.2 matplotlib=3.1.1 opencv=4.5.3 jupyter=1.0.0 holoviews=1.14.5 scipy=1.7.1 scikit-learn=0.24.2 bokeh=2.3.3 jinja2=3.0.3 tqdm
```
